### DISCUSSION TITLE
PCH file built from a different branch ((clang-1600.0.26.3)) than the compiler ((clang-1600.0.26.4))

### DISCUSSION THREAD

**Discussion by wyatt-wong** [mamedev/discussions#157] [Q&A] [answered]
After upgraded xcode from 16.0 to 16.1, I got the following eHrror when building mame under macOS Sequoia 15.1. How do I resolve this ?

Here is the script file I used to build mame:

`touch makefile`
`make REGENIE=1 TOOLS=1 DEPRECATED=0 -j5`

Clang 16.0.0 detected
Building configurations...
Running action 'gmake'...
Generating "build/projects/sdl/mame/gmake-osx-clang/Makefile"
Generating "build/projects/sdl/mame/gmake-osx-clang/qtdbg_sdl.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/osd_sdl.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ocore_sdl.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/utils.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/netlist.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/formats.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/expat.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/zlib.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/zstd.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/softfloat.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/softfloat3.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/jpeg.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/flac.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/7z.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/lua.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/lualibs.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sqlite3.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/portmidi.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/bx.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/bimg.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/bgfx.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/portaudio.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/linenoise.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/utf8proc.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/wdlfft.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ymfm.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/asmjit.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/emu.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/precompile.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/optional.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/dasm.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/frontend.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/pc.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/apple.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ampro.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/regnecentralen.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/technos.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/nasco.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/svi.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/intel.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ausnz.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/misc.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/mitsubishi.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/heathzenith.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/brother.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/kiwako.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/drc.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/appliedconcepts.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/cinematronics.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/next.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/elektron.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/pacman.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/nascom.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/thomson.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sun.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tch.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/elektor.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/barcrest.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tomy.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/cromemco.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/videoton.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/pacific.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/homelab.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/natsemi.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/midcoin.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/arcadia.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/gaelco.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/televideo.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/korg.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ussr.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/emusys.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ncd.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/memotech.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/interton.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/dg.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/amirix.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/vectorgraphic.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/nix.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/positron.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/nakajima.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/fuuki.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sgi.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/hp.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/booth.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/be.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ampex.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/unico.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tektronix.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/rolm.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/microkey.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/wicat.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/access.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/rockwell.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/wavemate.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/quantel.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/bfm.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/enterprise.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tecmo.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/mr.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/bitcorp.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/kaypro.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/irem.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/camputers.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/votrax.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/trs.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/universal.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/acorn.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/slicer.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/agat.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/yeno.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/fujitsu.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/gamepark.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/dgrm.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/epoch.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/osborne.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/verifone.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/mupid.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/morrow.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/hds.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tatung.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/cave.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sunwise.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/aristocrat.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/nmk.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/dai.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/conitec.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/skeleton.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/atari.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/excellent.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/olympia.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/microsoft.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/leapfrog.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/gridcomp.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/eaca.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/siemens.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/zvt.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/intergraph.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/phoenix.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/edevices.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/toshiba.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tiger.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/dynax.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/omnibyte.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/svision.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sequential.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/unisonic.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/burroughs.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ice.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/atlus.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tigertel.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/hec2hrp.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sega.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/openuni.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/comx.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/cce.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/stern.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/homebrew.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tesla.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/wang.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/wyse.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/samsung.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/vtech.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/comad.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/facit.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/mchester.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/f32.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/yamaha.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tangerine.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/commodore.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/stm.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/act.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/saturn.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/orla.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/shared.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/meadows.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/funworld.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/olympiaint.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/cirsa.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/nokia.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/galaxian.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/unicard.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/novation.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ensoniq.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tab.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ces.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sunelectronics.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/informer.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/nintendo.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/novag.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/metro.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/kyocera.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/dataeast.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/liberty.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tryom.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/canon.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/omori.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/amstrad.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/unisys.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ibm.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/adp.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/fairchild.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/usp.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tvgames.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/banctec.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sigma.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/chromatics.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/mips.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/swtpc.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/bandai.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/mg1.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ramtek.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/cantab.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/funtech.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/pce.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/thepit.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/seibu.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/westinghouse.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/rca.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tiki.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/multitech.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sinclair.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/igs.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/olivetti.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/qume.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/igt.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/subsino.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/excalibur.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/compugraphic.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/altos.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ericsson.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/miltonbradley.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/konami.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/koei.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/rare.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/makerbot.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/beehive.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/adc.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/cybiko.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/bondwell.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/microterm.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/apollo.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/upl.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/hegenerglaser.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/synertek.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/midway.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/yunsung.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/concept.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ultratec.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/psion.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/citoh.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/saitek.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ncr.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/visual.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/itech.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/exidy.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sord.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/venture.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/seta.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ultimachine.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/motorola.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/apf.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/virtual.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/snk.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/isc.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/robotron.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/fairlight.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/kontron.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ceres.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/dec.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/amiga.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/midw8080.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/falco.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/osi.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/mc.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/pitronics.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/promat.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/nec.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/samcoupe.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/akai.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/coleco.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tecfri.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/chess.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/msx.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/namco.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/microcraft.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/neogeo.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/toaplan.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/cvs.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/dooyong.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/eolith.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/kurzweil.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ddr.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/husky.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/roland.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/merit.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/netronics.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/astrocorp.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/omron.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tandberg.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/nichibutsu.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/handheld.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/alliedleisure.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/gottlieb.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/vsystem.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/novadesitec.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/zpa.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/bmc.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/matsushita.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/dms.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/kyber.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/att.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/playmark.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/luxor.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/xerox.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/benesse.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/zaccaria.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/psikyo.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/matic.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/philips.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/force.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/maygay.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/pinball.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/suna.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/valadon.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/teamconcepts.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/capcom.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/esprit.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/fidelity.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/telenova.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/cxg.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/kawai.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sealy.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/alpha.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/alba.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sanritsu.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ccs.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/kaneko.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/taito.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/jaleco.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/learsiegler.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/rm.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/telercas.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/hitachi.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ta.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/grundy.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/entex.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/wing.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sfrj.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/chessking.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/tatsumi.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/gametron.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/northstar.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sharp.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/adds.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/trainer.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/vidbrain.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/recfranco.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/alesis.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/mera.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/jpm.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sanyo.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/casio.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sage.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/mit.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/sony.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/orca.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/palm.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/poly88.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/mattel.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/lsi.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ti.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/efo.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/epson.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/mits.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/mame.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/romcmp.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/chdman.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/jedutil.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/unidasm.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ldresample.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/ldverify.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/regrep.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/srcclean.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/split.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/pngcmp.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/nltool.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/nlwav.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/castool.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/floptool.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/imgtool.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/aueffectutil.make"
Generating "build/projects/sdl/mame/gmake-osx-clang/testkeys.make"
Done. Generated 393/393 projects.
make -R --no-print-directory -C build/projects/sdl/mame/gmake-osx-clang config=release64 precompile
Compiling src/mame/benesse/challenge_gear.cpp...
error: PCH file built from a different branch ((clang-1600.0.26.3)) than the compiler ((clang-1600.0.26.4))
1 error generated.
make[2]: *** [benesse.make:344: ../../../../osx_clang/obj/x64/Release/src/mame/benesse/challenge_gear.o] Error 1
make[1]: *** [Makefile:958: benesse] Error 2
make[1]: *** Waiting for unfinished jobs....
Compiling src/lib/formats/all.cpp...
Archiving libformats.a...
make: *** [makefile:1349: macosx_x64_clang] Error 2

**Comment by rb6502** [comment, marked as answer]
Simply 'make clean' to clear the previously cached data.  Each time Xcode or its command-line tools are updated to a new version this error will occur the first time you build.

**Reply by wyatt-wong** [reply]
> Simply 'make clean' to clear the previously cached data. Each time Xcode or its command-line tools are updated to a new version this error will occur the first time you build.

Excellent, thanks very much. I thought I have to remove the build folder but I didn't want to do that because rebuild the whole source takes a couple of errors on my old MacBook Pro and so I only want to rebuild those files that were last changed from the source that I retrieved from GitHub.
